packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -11,8 +11,14 @@ var _ = require('lodash'),
     isSerializable = require('../lib/is-serializable.js').isSerializable,
     SplitPane = require('ubervu-react-split-pane'),
     localStorageLib = require('../lib/local-storage.js'),
+    CodeMirror = require('react-codemirror'),
     fuzzaldrinPlus = require('fuzzaldrin-plus');
 
+require('codemirror/lib/codemirror.css');
+require('codemirror/addon/fold/foldgutter.css');
+require('codemirror/theme/solarized.css');
```

Comment:
I agree with keeping this theme for customisation but can't we move the other .css required into the react-code-mirror fork (as we did with the .js requires). WDYT?